filtering_data.py

```
@@ -0,0 +1,73 @@
+            continue
+        if is_low_color_variance(image):
+            print(f"❌ 낮은 색상 다양성 이미지 제외: {img_name}")
+            filtered_out += 1
+            continue
+
+        output_path = os.path.join(output_folder, img_name)
+        cv2.imwrite(output_path, image)
+        saved_images += 1
+
+    # 최종 통계 출력
+    print(f"\n📊 처리 완료:")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)